asset/src/drop_field/schema.ts

```diff
@@ -0,0 +1,14 @@
+import { ConvictSchema } from '@terascope/job-components';
+import { DateRouterConfig } from '@terascope/standard-asset-apis';
+
+export default class Schema extends ConvictSchema<DateRouterConfig> {
+    build(): Record<string, any> {
+        return {
+            field: {
+                doc: 'Field to remove from incoming document',
+                default: null,
+                format: 'required_String'
+            }
```

Comment:
I was thinking about this processor accepting an array of field names.  Sparing myself multiple calls.  Or should it be a separate processor `drop_fields`?

Reply:
I think having a `drop_fields` and a `drop_field` may be too confusing.  I'll change the drop_field processor to accept a string or an array of strings